shared/teams/common/selection-popup.tsx

```diff
@@ -309,7 +367,7 @@ const ChannelMembersActions = ({conversationIDKey, teamID}: ChannelActionsProps)
         onClick={onAddToChannel}
         fullWidth={Styles.isMobile}
       />
-      <Kb.Button label="Edit role" mode="Secondary" onClick={onEditRoles} fullWidth={Styles.isMobile} />
+      <EditRoleButton teamID={teamID} members={members} />
```

Comment:
I think if you've selected any owner when you're an admin we should remove or disable this, maybe with a tooltip.

Reply:
Nice, I like this. Disabled the button whenever there are no changes allowed.

Reply:
Cool. It doesn't show on tap on mobile. I think tooltip has a prop for that.